Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -956,14 +956,27 @@ static inline ASPublicKeyCredentialResidentKeyPreference toASCResidentKeyPrefere
     return requestContext;
 }
 
-static Vector<WebCore::AuthenticatorTransport> toAuthenticatorTransports(NSArray<NSNumber *> *ascTransports)
+static Vector<WebCore::AuthenticatorTransport> toAuthenticatorTransports(NSArray<NSString *> *ascTransports)
 {
     Vector<WebCore::AuthenticatorTransport> transports;
     transports.reserveInitialCapacity(ascTransports.count);
-    for (NSNumber *ascTransport : ascTransports) {
-        if (WTF::isValidEnum<WebCore::AuthenticatorTransport>(ascTransport.intValue))
-            transports.append(static_cast<WebCore::AuthenticatorTransport>(ascTransport.intValue));
+    for (NSString *ascTransport : ascTransports) {
+        if ([ascTransport isEqualToString:@"usb"])
```

Comment:
I'm not in love with this loop; I think it would be clearer if you had a static function that returned a `WebCore::AuthenticatorTransport` enum value when presented with an `NSString *` argument, then the loop would be very simple. But this is okay as-is.

Reply:
I figured we need this lump of if-elses somewhere, and it's only used in this one spot so pulling it out to be its own function wouldn't really give much benefit.

Reply:
We typically do this as a form of documentation. But I think this is fine.

Reply:
I agree with Brent in moving the mapping from string to enum to a function.